src/ontology/mondo-edit.obo

```diff
@@ -424775,7 +424776,8 @@ synonym: "puerperal disorder" EXACT [MESH:D011644]
 xref: MESH:D011644 {source="MONDO:equivalentTo"}
 xref: SCTID:362973001 {source="MONDO:equivalentTo"}
 xref: UMLS:C0034040 {source="MONDO:equivalentTo"}
-is_a: MONDO:0000001 {source="https://github.com/monarch-initiative/mondo/issues/652"} ! disease or disorder
+is_a: MONDO:0700003 ! obstetric disorder
```

Comment:
could you please add a source annotation here
you can add the ticket https://github.com/monarch-initiative/mondo/issues/3161 and your ORCID